service/core-fuzzers.yaml

```diff
@@ -16,3 +16,4 @@ fuzzers:
     - libfuzzer
```

Comment:
I'm getting confused what the purpose of this file is.
Is it supposed to include every fuzzer that's going to be included in fuzzbench on a permanent basis?

Reply:
This is used for deciding which benchmarks are the standard ones to be included/added/merged with a new experiment.

Reply:
We will also use this to decide which fuzzers are variants and which ones are not. This will be used to seperate official experiment reports in root dir and variant experiment reports in experimental sub-dir. @lszekeres is working on this.